subprojects/core-api/src/main/java/org/gradle/api/file/DuplicatesStrategy.java

```diff
@@ -53,7 +53,13 @@
      * <p> 
      * Use this strategy when duplicates are an error condition that should cause the build to fail.
      */
-    FAIL
-
+    FAIL,
 
+    /**
+     * The default strategy, which is inherited from the parent copy spec, if any,
+     * or {@link DuplicatesStrategy#INCLUDE} if the copy spec has no parent.
+     *
+     * @since 5.0
+     */
+    DEFAULT
```

Comment:
I think `INHERIT` would be a better name.

Reply:
Definitely. Thanks.